### PR TITLE
Update Sohu.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Sohu.com.xml [hunk 1]
```diff
@@ -40,80 +36,51 @@
 		- login.mail ²
 		- mgame ²
 		- money ²
-		- news ²
 		- photocdn ²
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Sohu.com.xml]: Different than http

File: src/chrome/content/rules/Sohu.com.xml [hunk 2]
```diff
@@ -40,80 +36,51 @@
...
-	<!--	Direct rewrites:
-				-->
+	<target host="txt.go.sohu.com" />
+		<test url="http://txt.go.sohu.com/ip/soip" />
 	<target host="images.sohu.com" />
+		<test url="http://images.sohu.com/bill/s2015/yanchunzhang/richan/logo.png" />
+	<target host="js.sohu.com" />
 	<target host="mail.sohu.com" />
 	<target host="passport.sohu.com" />
 	<target host="pay.sohu.com" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Sohu.com.xml]: https://pb.sogou.com/pb.js

File: src/chrome/content/rules/Sohu.com.xml [hunk 3]
```diff
@@ -26,9 +24,7 @@
 
 
 	Problematic hosts in *sohu.com:
-
 		- ^ ¹
-		- ad ²
 		- auto ²
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Sohu.com.xml]: Remove this

File: src/chrome/content/rules/Sohu.com.xml [hunk 4]
```diff
@@ -40,80 +36,51 @@
...
-
-			- tv from b.scorecardresearch.com *
-			- tv from sohu.wrating.com *
-
-		- favicon on passport from s0.life.itc.cn *
-
-	* Secured by us, see https://www.paulirish.com/2010/the-protocol-relative-url/
-
+	MCB:
+		- ad.sohu.com
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Sohu.com.xml]: auto.sohu.com

File: src/chrome/content/rules/Sohu.com.xml [hunk 5]
```diff
@@ -40,80 +36,51 @@
...
-		- Bugs, on:
-
-			- tv from b.scorecardresearch.com *
-			- tv from sohu.wrating.com *
-
-		- favicon on passport from s0.life.itc.cn *
-
-	* Secured by us, see https://www.paulirish.com/2010/the-protocol-relative-url/
-
+	MCB:
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Sohu.com.xml]: I won't ask you do it now, but some of these can be secured if you have time.

**Comment by gloomy-ghost** [comment]
```
ERROR src/chrome/content/rules/Sohu.com.xml: Fetch error: http://pb.sohu.com/ => https://pb.sohu.com/: (6, 'Could not resolve host: pb.sohu.com')
ERROR src/chrome/content/rules/Sohu.com.xml: Fetch error: http://ushq.stock.sohu.com/ => https://ushq.stock.sohu.com/: (35, 'Unknown SSL protocol error in connection to ushq.stock.sohu.com:443 ')
```

这都是什么奇怪的错误……

**Comment by gloomy-ghost** [comment]
你给的两个链接我都可以打开，用 https://photocdn.sohu.com/20060930/Img227840979.swf 吧

**Comment by gloomy-ghost** [comment]
擦，我的锅，`pb.sohu.com` 真的不在DNS里……搜狗搜狐傻傻分不清

**Comment by ivysrono** [comment]
擦，我也没看……

**Pull request merged**